cms/admin/placeholderadmin.py

```diff
@@ -609,6 +643,7 @@ def move_plugin(self, request):
         target_placeholder_admin = self._get_attached_admin(placeholder)
 
         if move_a_copy:  # "paste"
+            plugins = [plugin] + list(plugin.get_descendants())
```

Comment:
any reason not to use ``plugin.get_tree()``? this will return a queryset with all descendants and the current plugin